redis/utils.go

```diff
@@ -1,7 +1,58 @@
 package redis
 
-import "fmt"
+import (
+	"context"
+	"github.com/cenkalti/backoff/v4"
+	"github.com/dora-network/dora-service-utils/orderbook"
+	"github.com/redis/go-redis/v9"
+	"strings"
+)
 
+// SequenceNumberKey returns the key for the sequence number of a user.
 func SequenceNumberKey(userID string) string {
-	return fmt.Sprintf("%s:%s", SequenceNumberPrefix, userID)
+	return Key(SequenceNumberPrefix, userID)
+}
+
+// OrderBookOrdersKey returns the key for retrieving an order book's open orders.
+func OrderBookOrdersKey(orderBookID string) string {
+	return Key(OrderBookPrefix, orderBookID)
```

Comment:
I think we will most likely want to save orders with different state in different tables as we do on spanner now. But this could be extended in the future. cc @toteki

Reply:
The Redis cache should only be storing live orders, e.g. Open or partially filled, Spanner will store all order information and would be used for any API requests. The Redis cache is only for operating on live order information for match making so that we can scale the order book service if we need to.